CRM/Event/WorkflowMessage/EventExamples.php

```diff
@@ -61,6 +61,7 @@ public function build(array &$example): void {
   private function addExampleData(GenericWorkflowMessage $messageTemplate, $example): void {
     $messageTemplate->setContact(\Civi\Test::example('entity/Contact/Barb'));
     $messageTemplate->setEventID($example['event_id']);
+    $messageTemplate->setContribution(['total_amount' => 50, 'balance_amount' => 20, 'currency' => 'USD']);
```

Comment:
Actually I'm going to do a bit more work on the example -  this doesn't really show up as yet in the preview